pages/common/postmap.md

```diff
@@ -0,0 +1,24 @@
+# postmap
+
+> Create or query one or more Postfix lookup tables, or update an existing one.
+> More information: <https://manned.org/postmap>.
+
+- Search specified maps for the key and remove one entry per map:
+
+`postmap -d {{key}} {{input_file}}`
```

Comment:
We're currently in the process of mandating that all paths start with `path/to/`.

```suggestion
`postmap -d {{key}} {{path/to/file}}`
```